scripts/source/source_cloud_native_operator.py

```diff
@@ -37,42 +37,62 @@ def index_frontmatter():
                 readingNav = True
             elif readingNav:
                 nav.append(line.replace(".md", ""))
+                if "quickstart.md" in line:
+                    nav.append("  - interactive\n")
 
     return INDEX_FRONTMATTER.format("".join(nav))
 
 
 def process_md(file_path):
-    new_file_path = file_path.replace(".md", ".mdx")
+    new_file_path = file_path.with_suffix(".mdx")
 
     with open(new_file_path, "w") as new_file:
         with open(file_path, "r") as md_file:
             copying = False
-            gh_relative_path = file_path.replace("temp_kubernetes/build/", "src/")
+            quickstart = file_path.name == "quickstart.md"
+            paragraph = 0
+            gh_relative_path = "src/" + str(
+                file_path.relative_to("temp_kubernetes/build/")
+            )
 
             for line in md_file:
-                if copying:
-                    new_file.write(rewrite_yaml_links(line))
-                if line.startswith("#") and not copying:
+                if not line.strip():
```

Comment:
[q] We can shortcut based on `quickstart` right here, since it's the only file we care about.